docs/source/irene.rst

```diff
@@ -161,7 +161,8 @@ Peak finding and matching of PMT and SiPM signals
 
 The peak finding and waveform slicing is arguably the most complex part of the **RWF** processing. The algorithm must be able to find two very different types of signals (S1 and S2), while accurately establishing the limits on those peaks to maintain the energy resolution capabilities of the detector.
 
-In order to optimize the peak search, PMT **CCWF**\ s are used as they have a higher sampling rate and therefore better time resolution. On top of that, these waveforms are PMT-summed to increase the signal-over-noise ratio [#]_. S1 and S2 signals are searched independently.
+In order to optimize the peak search, PMT **CCWF**\ s are used as they have a higher sampling rate and therefore better time resolution. The searches for S1 and S2 signals use slightly different waveforms. The PMT waveforms contain some low-frequency noise that can shift the PMT baseline locally by a small amount. The MAU (Moving Average Uindow) accounts for these small local variations of the baseline, allowing for a higher peak finding efficiency. The parameter ``n_mau`` controls the size of this window. The S1 peak search is performed by looking for samples of the PMT waveform that deviate from the MAU-generated baseline more than ``thr_mau``. On the other hand, the S2 peak search uses the **CCWF**\ s directly, without a MAU.
```

Comment:
What is a `Uindow`? Is it a spelling typo or is it on purpose?

Reply:
The acronym is MAW, but it's spelled MAU in the code.

Reply:
I see. It's a bit confusing, how about changing the code to the correct name `maw`?

Reply:
we can do that. Maybe there was a reason to call it mau, but I'm not aware of it.